libs/blocks/chart/chart.js

```diff
@@ -189,6 +190,7 @@ const handleIntersect = (chart, chartOptions) => (entries, observer) => {
 
   entries.forEach((entry) => {
     if (entry.isIntersecting) {
+      chart?.resize();
```

Comment:
Why is the chart resizing before the options are even set? There's no chart to resize

Reply:
The chart is created before the observer, flexbox is changing the width so I have to resize. 
If I resize after setting the options then the animation gets canceled.

Reply:
Ah, in that case, it seems like we shouldn't be initializing the chart ahead of time. Instead do the init in the observer handler